src/include/com/amazonaws/kinesis/video/common/Include.h

```diff
@@ -50,6 +50,8 @@ extern "C" {
 #define STATUS_FILE_CREDENTIAL_PROVIDER_OPEN_FILE_FAILED    STATUS_COMMON_PRODUCER_BASE + 0x00000022
 #define STATUS_FILE_CREDENTIAL_PROVIDER_INVALID_FILE_LENGTH STATUS_COMMON_PRODUCER_BASE + 0x00000023
 #define STATUS_FILE_CREDENTIAL_PROVIDER_INVALID_FILE_FORMAT STATUS_COMMON_PRODUCER_BASE + 0x00000024
+#define STATUS_ECS_FAILED                                   STATUS_COMMON_PRODUCER_BASE + 0x00000025

```

Comment:
Make it a little more descriptive - something like STATUS_ECS_AUTH_FAILED or something

Reply:
I separated it into three error codes.